src/XMakeTasks/ManifestUtil/Strings.resx

```diff
@@ -145,7 +145,7 @@
     <value xml:space="preserve">Warning while signing {0}. {1}</value>
   </data>
   <data name="SecurityUtil.SigntoolNotFound">
-    <value xml:space="preserve">SignTool.exe not found.</value>
+    <value xml:space="preserve">SignTool.exe not found. Path: {0}</value>
```

Comment:
No, but you're right that it should probably have *some* punctuation. Looks like we'd match the pattern in this file with

```
SignTool.exe was not found at path {0}.
```